### ISSUE TITLE
Wrong translation component designation

### ISSUE THREAD

**Issue by comradekingu** [dsheeler/CoverflowAltTab#214]
The main component should not be "dictionary", as that makes it much harder to translate them.
Can be changed in https://hosted.weblate.org/settings/coverflow-alt-tab/source/
:)

**Comment by dsheeler** [comment]
Thank you so much for your help already!  Amazing!  I'm totally new to weblate. Is this the same thing as the "glossary"? I'm still figuring this out!

**Comment by comradekingu** [comment]
@dsheeler 
Yes, it is meant for something that contributes to one of the components.
Where did the project on WL go?

**Comment by dsheeler** [comment]
@comradekingu https://hosted.weblate.org/projects/coverflow-alt-tab/ does that not work for you? Can you tell me if it's correct now???

**Comment by comradekingu** [comment]
Ah, it was me testing the link I had made in the post above with /settings/, which doesn't work for me.
You can add "kingu" if you want me to have a look at the different things. :)

**Comment by dsheeler** [comment]
@comradekingu, I added you as an admin. Thanks. I definitely want this configured correctly so people can more easily contribute!
